client/components/data/store-connection/index.jsx

```diff
@@ -70,6 +76,12 @@ class StoreConnection extends React.Component {
 			return React.createElement( this.props.component, this.state );
 		}
 
+		if ( 'right' === abtest( 'showCheckoutCartRight' ) ) {
+			return React.Children.map( this.props.children, child => {
```

Comment:
By way of explaining, we're removing the restriction of having only one child, and instead iterating through all children because moving the cart to the right makes `<CheckoutData>` [to have two children](https://github.com/Automattic/wp-calypso/pull/32832/files#diff-7f164a3b7deeca0666c8602e2b20e4acR49).

Reply:
I think it is a good idea to make this higher-level component more general, as I don't see why it should only be functional for one child. There is no need to change it now, just wanted to note that once we have concluded the checkout cart on the right test, I'd suggest that we keep it this way.